experimental/bindings/java/com/google/iree/native/context_wrapper.cc

```diff
@@ -104,8 +104,9 @@ Status ContextWrapper::InvokeFunction(const FunctionWrapper& function_wrapper,
     // Wrap the input buffers in buffer views.
     iree_hal_buffer_view_t* input_buffer_view = nullptr;
     IREE_RETURN_IF_ERROR(iree_hal_buffer_view_create(
-        input_buffer, /*shape=*/&input_element_count,
-        /*shape_rank=*/1, IREE_HAL_ELEMENT_TYPE_FLOAT_32,
+        input_buffer,      IREE_HAL_ELEMENT_TYPE_FLOAT_32,
+ /*shape=*/&input_element_count,
+        /*shape_rank=*/1, 
```

Comment:
run clang-format please

we could enable the checks now:
https://github.com/google/iree/blob/d496bc9d53e918c0362ddd498ded750eba6050f3/experimental/bindings/java/.clang-format#L15-L17

Reply:
If you use clang-format `style=file` (which I recommend), this `.clang-format` file with `DisableFormat` will turn it off. We had it off since I think it was running on the Java files, which differed from our internal Java formatting. If enough changes will be made in a directory like this we should re-evaluate (as this comment says) and find a setup that works well for C++ and Java source code across both source repositories.

Reply:
I ran clang format after nullifying the disable. That fixed the cc file. Since we're still not sure about the ramifications, I'll leave the `DisableFormat` alone for now.